gulpfile.js

```diff
@@ -38,10 +38,10 @@ gulp.task('styles', gulp.series(
 
 /**
  * Copy assets task
- * Copies assets to taskArguments.destination (public)
+ * Copies assets to taskArguments.destination (dist)
  */
 gulp.task('copy:assets', () => {
-  return gulp.src(`${configPaths.src}assets/**/*`)
+  return gulp.src(slash(join(configPaths.assets, '**/*')))
```

Comment:
See commit https://github.com/alphagov/govuk-frontend/commit/6e724719f3a9dc4cccf18f9f77d7ea3ae2eabfdf regarding [Gulp using its own glob library](https://github.com/gulpjs/glob-parent#windows):

>**Note:** Gulp globs use `slash()` to ensure Windows paths are flipped back to POSIX for compatibility

I did consider `require('path/posix').normalize` to flip Windows backslashes back to POSIX forward slashes but we'll continue to use `slash()` for this instead